exercises/storeWithObjects.js

```diff
@@ -74,6 +66,7 @@ for (let user of allUsers) {
   if (userCanAffordSofa(user, sofa)) {
     console.log('Congratulations!');
     console.log(`A ${sofa['name']} costs ${sofa['price']}, which is within your budget of ${user['budget']}!`);
+    console.log('Confirmation has been sent to your email');
```

Comment:
Include the email address itself in the message. Given a particular user, how do you access their email address?